carbonmark/pages/retirements/[beneficiary]/index.tsx

```diff
@@ -58,7 +59,14 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (ctx) => {
 
       try {
         totalCarbonRetired = klimaRetires
-          .reduce((acc, retirement) => acc + parseFloat(retirement.amount), 0)
+          .reduce((acc, retirement) => {
+            const amount = retirement.retire.credit.project.registry.startsWith(
+              "ICR"
+            )
+              ? retirement.retire.amount
+              : formatUnits(retirement.retire.amount);
```

Comment:
eww.. lol

Reply:
Very open to any suggestions on this! There is some refactoring coming down the pipeline that will aim to standardize registry handling by the time it reaches the api or app. Right now the subgraph handles all amounts in wei; it probably makes sense to move this handling into that layer.